moulinette/actionsmap.py

```diff
@@ -368,17 +368,23 @@ def __init__(self, parser_class, namespaces=[], use_cache=True,
         for n in namespaces:
             logger.debug("loading actions map namespace '%s'", n)
 
-            if use_cache:
+            actionsmap_yml = '%s/actionsmap/%s.yml' % (pkg.datadir, n)
+            actionsmap_pkl = '%s/actionsmap/%s-%s.pkl' % (pkg.cachedir, n, os.stat(actionsmap_yml).st_size)
```

Comment:
I think situation with only characters change is more than 0,1%. I think use st_mtime in more could be a good idea. For the moment I had not test this code on a ynh instance, but I will and add this small change.